src/php/settings/version-switch.php

```diff
@@ -0,0 +1,579 @@
+					<?php esc_html_e( 'Switch Version', 'code-snippets' ); ?>
+				</button>
+			</p>
+
+			<div id="version-switch-result" class="notice" style="display: none;"></div>
+		<?php endif; ?>
+	</div>
+
+	<script type="text/javascript">
+	jQuery(document).ready(function($) {
+		var currentVersion = '<?php echo esc_js( $current_version ); ?>';
+		var $button = $('#switch-version-btn');
+		var $dropdown = $('#target_version');
+		var $result = $('#version-switch-result');
+		
+		// Handle dropdown changes - enable/disable button and show/hide warning
+		$dropdown.on('change', function() {
+			var selectedVersion = $(this).val();
+			
+			if (!selectedVersion || selectedVersion === currentVersion) {
+				// Current version or no selection - disable button and hide warning
+				$button.prop('disabled', true);
+				$('#version-switch-warning').hide();
+			} else {
+				// Different version selected - enable button and show warning
+				$button.prop('disabled', false);
+				$('#version-switch-warning').show();
+			}
+		});
+		
+		$button.on('click', function() {
+			var targetVersion = $dropdown.val();
+			
+			if (!targetVersion || targetVersion === currentVersion) {
+				$result.removeClass('notice-success notice-error').addClass('notice-warning')
+					.html('<p><?php esc_html_e( 'Please select a different version to switch to.', 'code-snippets' ); ?></p>')
+					.show();
+				return;
+			}
+
+			// Disable button and show loading
+			$button.prop('disabled', true).text('<?php esc_html_e( 'Switching...', 'code-snippets' ); ?>');
+			$result.removeClass('notice-success notice-error notice-warning').addClass('notice-info')
+				.html('<p><?php esc_html_e( 'Processing version switch. Please wait...', 'code-snippets' ); ?></p>')
+				.show();
+
+			// Make AJAX request
+			$.post(ajaxurl, {
+				action: 'code_snippets_switch_version',
+				target_version: targetVersion,
+				nonce: '<?php echo esc_js( wp_create_nonce( 'code_snippets_version_switch' ) ); ?>'
+			})
+			.done(function(response) {
+				if (response.success) {
+					$result.removeClass('notice-info notice-error').addClass('notice-success')
+						.html('<p>' + response.data.message + '</p>');
+					
+					// Refresh page after 3 seconds
+					setTimeout(function() {
+						window.location.reload();
+					}, 3000);
+				} else {
+					$result.removeClass('notice-info notice-success').addClass('notice-error')
+						.html('<p>' + response.data.message + '</p>');
+					$button.prop('disabled', false).text('<?php esc_html_e( 'Switch Version', 'code-snippets' ); ?>');
+				}
+			})
+			.fail(function() {
+				$result.removeClass('notice-info notice-success').addClass('notice-error')
+					.html('<p><?php esc_html_e( 'An error occurred while switching versions. Please try again.', 'code-snippets' ); ?></p>');
+				$button.prop('disabled', false).text('<?php esc_html_e( 'Switch Version', 'code-snippets' ); ?>');
+			});
+		});
+	});
+	</script>
```

Comment:
- Can we rather add this script to a separate JS file and enqueue it?
- Do we have to use jQuery?
- Use `let/const` instead of `var`